app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -34,33 +34,46 @@ class PlatformInitiativesService @Inject()(
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
   def allPlatformInitiatives(team: Option[String] = None)(implicit ec: ExecutionContext): Future[Seq[PlatformInitiative]] = {
+    println(team)
     val initiatives = Seq(
       createDefaultBranchInitiative(
         initiativeName        = "Update Default Branch Terminology",
-        initiativeDescription = "To update default branch names - [Default branch tracker](https://catalogue.tax.service.gov.uk/defaultbranch).",
+        initiativeDescription = team match {
+          case None       => "To update default branch names - [Default Branch Tracker](https://catalogue.tax.service.gov.uk/defaultbranch?name=&teamNames=&defaultBranch=master) | [Confluence](https://confluence.tools.tax.service.gov.uk/display/TEC/2021/10/08/Default+Branch+Migration%3A+How+To)."
+          case Some(team) => s"To update default branch names - [Default Branch Tracker for $team](https://catalogue.tax.service.gov.uk/defaultbranch?name=&teamNames=$team&defaultBranch=master) | [Confluence](https://confluence.tools.tax.service.gov.uk/display/TEC/2021/10/08/Default+Branch+Migration%3A+How+To)."
```

Comment:
OBSERVATION: Could we put these into the messages resources?

Reply:
Updated to remove the duplication of descriptions